src/libstore/local.mk

```diff
@@ -37,24 +40,35 @@ ifdef HOST_UNIX
   INCLUDE_libstore += -I $(d)/unix
 endif
 
+ifdef HOST_WINDOWS
+NIX_ROOT = N:\\\\
```

Comment:
This seems like an important choice. Does this mean that the standard Nix store is `N:\nix\store`?

Reply:
I suppose it does for now, but I consider this just a non-decision just temporary move  to make the unit tests runable. (If those paths are not valid Windows paths on Windows, global constructors relating to settings will fail, for example.)

Reply:
This is, however, already in a separate commit because it is a matter of making Nix run on Windows, rather than merely just build for Windows, so I could imagine saving it for a later PR if you like.